dotcom-rendering/src/layouts/PictureLayout.tsx

```diff
@@ -408,17 +412,17 @@ export const PictureLayout = (props: WebProps | AppsProps) => {
 				lang={decideLanguage(article.lang)}
 				dir={decideLanguageDirection(article.isRightToLeftLang)}
 			>
-				{renderingTarget === 'Apps' && (
+				{isApps && (
 					<Island priority="critical" clientOnly={true}>
 						<AdPortals />
 					</Island>
 				)}
 				<Section
 					fullWidth={true}
 					showTopBorder={false}
-					backgroundColour={palette.background.article}
+					backgroundColour={themePalette('--article-background')}
 					element="article"
-					borderColour={palette.border.secondary}
+					borderColour={themePalette('--article-border')}
```

Comment:
I think this should be 
```suggestion
					borderColour={themePalette('--article-border-secondary')}
```